src/config.rs

```diff
@@ -302,7 +302,7 @@ impl Default for Config {
             sentry_dsn: None,
             caches: CacheConfigs::default(),
             symstore_proxy: true,
-            sources: Arc::new(vec![]),
+            sources: Arc::from(vec![]),
```

Comment:
even though this does not allocate, I wonder if there is a simpler way?

Reply:
Probably `Arc::from(&[])`

Reply:
Nope, you need to get a slice but getting a slice from an array is unstable.